src/pages/yujin/Main/FeedsPage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Feeds from './Feeds';
+
+function FeedsPage() {
+  return (
+    <div>
+      <Feeds />
+    </div>
+  );
+}
```

Comment:
how about
```suggestion
function FeedsPage() {
  return <Feeds />;
}
```

div는 없어도 좋을 것 같습니다.